pkg/profilecreator/profilecreator.go

```diff
@@ -445,3 +453,36 @@ func ensureSameTopology(topology1, topology2 *topology.Info) error {
 
 	return nil
 }
+
+// GetAdditionalKernelArgs returns a set of kernel parameters based on the power mode
```

Comment:
but the API contract doesn't guarantee to return a sorted list (which is fine but then in the tests we should not assume that)